languages/swift/exercises/concept/dictionaries/.docs/introduction.md

```diff
@@ -0,0 +1,66 @@
+[Dictionaries][dictionaries] are one of Swift's three primary collection types. Dictionaries store mappings between _keys_ which are elements of one type and _values_ which are elements of another type (possibly the same type as that of the keys.
```

Comment:
Yeah. I believe I'd seen links in a couple of other exercises so I added them, but I agree that it can lead a student down a rabbit hole. I'll pull them.